Mathlib/Algebra/Lie/Submodule.lean

```diff
@@ -106,6 +106,15 @@ theorem mem_coe {x : M} : x ∈ (N : Set M) ↔ x ∈ N :=
 protected theorem zero_mem : (0 : M) ∈ N :=
   zero_mem N
 
+protected theorem add_mem {x y : M} : x ∈ N → y ∈ N → (x + y : M) ∈ N :=
+  add_mem
+
+protected theorem sub_mem {x y : M} : x ∈ N → y ∈ N → (x - y : M) ∈ N :=
+  sub_mem
+
+theorem smul_mem (t : R) {x : M} (h : x ∈ N) : t • x ∈ N :=
+  (N : Submodule R M).smul_mem t h
```

Comment:
We don't need these because we have already registered instances of `AddMemClass` and `SMulMemClass`:
```suggestion
```
You can witness this via:
```lean
#synth AddMemClass (LieSubmodule R L M) M
#synth AddSubgroupClass (LieSubmodule R L M) M -- Stronger form (covers negation)
#synth SMulMemClass (LieSubmodule R L M) R M
```